chart/validator-plugin-oci/values.yaml

```diff
@@ -58,5 +58,7 @@ env:
   #   value: http://192.168.1.100:8080
   # - name: NO_PROXY
   #   value: foo.bar.com,127.0.0.1
-# Optionally specify the name of the secret containing the proxy ca certificate
-# proxySecretName: proxy-cert
+proxy:
+  image: ubuntu
```

Comment:
I'm a fan of collecting all proxy-related configuration under `Values.proxy`, but the UX becomes a bit awkward when everything is conditional on `secretName` being specified and that isn't clearly outlined in the comments.

I would suggest adding `proxy.enabled`, which defaults as `false`, to clarify the UX. You can then make `proxy.secretName` required because the blocks where it is used will be wrapped in a conditional that's only entered when `proxy.enabled == true`.